src/main/app/form/Form.ts

```diff
@@ -0,0 +1,13 @@
+
+export class Form<T> {
+
+  public getErrors(body: T): FormError[] {
+    return [];
```

Comment:
i assume this is going to return something else later on?

Reply:
yep I'll make another branch for the form validation